uint32 RespondingNode
uint8  AntennaMode
uint8  rangeStatus
uint32 PRM
uint32 CRE
uint32 FRE
uint32 PRMError
uint32 CREError
uint32 FREError
uint8  rangeType

bool busy
bool failed

int32 msgID
